gpt_2_simple/gpt_2.py

```diff
@@ -348,12 +345,12 @@ def sample_batch():
                             avg_loss[1] * 0.99 + 1.0)
 
                 print(
-                    '[{counter} | {time:2.2f}] loss={loss:2.2f} avg={avg:2.2f}'
+                    '[{} | {2.2f}] loss={2.2f} avg={2.2f}'
                     .format(
-                        counter=counter,
-                        time=time.time() - start_time,
-                        loss=v_loss,
-                        avg=avg_loss[0] / avg_loss[1]))
+                        counter,
+                        time.time() - start_time,
+                        v_loss,
+                        avg_loss[0] / avg_loss[1]))
```

Comment:
Really no particular reason here, other than the fact that the identifiers were unnecessary (and personal preference).